firstvoices/backend/tasks/import_job_tasks.py

```diff
@@ -11,6 +11,76 @@
 )
 from backend.resources.dictionary import DictionaryEntryResource
 
+VALID_HEADERS = [
+    "title",
+    "type",
+    "translation",
+    "audio",
+    "image",
+    "video",
+    "video_embed_link",
+    "category",
+    "note",
+    "acknowledgement",
+    "part_of_speech",
+    "pronunciation",
+    "alt_spelling",
+    "visibility",
+    "include_on_kids_site",
+    "include_in_games",
+    "related_entry",
+]
+
+
+def is_valid_header_variation(input_header):
+    # The input header can have a _n variation from 2 to 5, e.g. note_5
+    # All other variations are invalid
+
+    splits = input_header.split("_")
+    if len(splits) >= 2:
+        prefix = "_".join(splits[:-1])
+        variation = splits[-1]
+    else:
+        prefix = input_header
+        variation = None
+
+    # Check if the prefix is a valid header
+    if prefix in VALID_HEADERS and variation and variation.isdigit():
+        variation = int(variation)
+        if variation < 1 or variation > 5:
```

Comment:
It's not allowed to be 1 either, right? This should be <=1

Reply:
Oh right, nice catch!